client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py

```diff
@@ -62,18 +62,19 @@ def process(self, instance):
         # If there's no data to set/update, skip creation of SG version
         if not data_to_update:
             self.log.info(
-                "No data to integrate to SG for subset '%s', skipping version creation.",
-                instance.data["subset"]
+                "No data to integrate to SG for product name "
+                f"'{instance.data['productName']}', skipping "
+                "version creation."
             )
             return
-        
+
         sg_session = context.data["shotgridSession"]
 
         # TODO: Use path template solver to build version code from settings
         anatomy = instance.data.get("anatomyData", {})
```

Comment:
Note for some future cleanup PR, this variable should not be called `anatomy`.

Reply:
fixed in f9ddbb37e370bbd1b6c7073afe4ad385858d0fe4